src/sortStudents.ts

```diff
@@ -1,16 +1,73 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAverageGrade(student: Student): number {
+  const averageGrade: number = student.grades.reduce(
+    (a: number, b: number) => a + b, 0,
+  )
+    / student.grades.length;
```

Comment:
Why take the entire student object as an argument if we only want the grades array?
What are the names for the reduce method arguments?

```suggestion
function getAverageGrade(grades: number[]): number {
  return grades.reduce((acc, grade) => (
    acc + grade), 0) / grades.length;
```